src/lib/pages/frontsite/Documents/DocumentDetails/DocumentCirculation/LoanAvailability.js

```diff
@@ -47,7 +47,9 @@ export class LoanAvailability extends Component {
         <List.Item>
           <List.Icon name="info" />
           <List.Content>
-            Long waiting queue. Contact the library for more information.
+            This paper book is currently not available in the Library. If you
+            need this book, please place a request on it, we will do our best to
+            provide you with the book as soon as possible.
```

Comment:
```suggestion
            The physical copy of this literature is currently not available in the Library. If you
            need a copy, please place a request on it, we will do our best to
            provide you with the literature as soon as possible.
```

Reply:
@KonstantinaStoikou @kprzerwa this change was not taken into account?
I think it should be something like:
```
There are no physical copies for this literature currently available at the library. If you would like to loan it, please place a request. The library staff will evaluate it.
```

The `The physical copy ` is not correct because we are in the Document details page.
The `we will do our best to provide you with the literature as soon as possible.` looks to me a CERN policy and therefore should not go in ILS.

This component should be overridable so the text can be easily changed.